CRADLE/CallPeak/callPeak.py

```diff
@@ -25,7 +22,6 @@ def mergePeaks(peak_result):
 	merged_peak = []
 
 	pastChromo = peak_result[0][0]
-	pastStart = int(peak_result[0][1])
```

Comment:
@Young-Sook I removed this variable because it's only assigned to, it's never read from.